CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.3.0-wip
+
+- Added support for lightweight tags.
```

Comment:
@Stargator – I did this for you. For future: pubspec and changelog entries!!

Reply:
Thanks, I also saw the changes to the test. I didn't know there was an hasLength matcher, so good to know.

This all looks good to me.